code/modules/projectiles/ammunition/ammo_casings.dm

```diff
@@ -250,6 +250,43 @@
 	caliber = "75"
 	projectile_type = "/obj/item/projectile/bullet/gyro"
 
```

Comment:
For above, you need to implement the caseless icon update changes that were in the foam force PR.